src/test/resources/io/jenkins/plugins/analysis/warnings/ajc.txt

```diff
@@ -0,0 +1,54 @@
+[INFO] ------------------------------------------------------------------------
+[INFO] Building Unnamed - com.company.products:project-ejb:ejb:6.1.0-SNAPSHOT
+[INFO]    task-segment: [clean, deploy]
+[INFO] ------------------------------------------------------------------------
+[INFO] [clean:clean {execution: default-clean}]
+[INFO] Deleting directory /home/hudson/workspace/project/project-ejb/target
+[INFO] [enforcer:enforce {execution: enforce-banned-dependencies}]
+-----------------------------------------------------
+[INFO] [aspectj:compile {execution: default}]
+[INFO] Showing AJC message detail for messages of types: [error, warning, fail]
+[WARNING] incorrect classpath: /home/hudson/.m2/repository/org/apache/cxf/cxf/2.6.1/cxf-2.6.1.pom
+Missing message: configure.invalidClasspathSection in: org.aspectj.ajdt.ajc.messages
+	<unknown source file>:<no line information>
```

Comment:
`Paths.get("<unknown source file>")` schlägt unter Windows fehl, da `<` nicht im Pfad vorkommen darf.

Reply:
Wo wird denn Paths.get aufgerufen?

Reply:
Ich habe keine Info ob das tatsächlich so vorkommt (der Parser kommt von einem PR). Aber ich denke mal, sicherheitshalber sollte man das <unknown source file> tatsächlich durch ein - oder ähnliches ersetzen. (Der Parser insgesamt scheint nicht gerade die beste Code Qualität zu haben ;-)

Reply:
@uhafner Der Parser erstellt ein `Issue` mit ungültigem Dateinamen. Später schlägt das Erstellen des Fingerprints fehlt, wenn versucht wird den Dateinamen in einen Path zu übersetzten:
bei `FullTextFingerprint#readLinesFromFile`

Scheinbar gibt der AspectJ Compiler tatsächlich diese Zeilen aus. 

(https://www.google.de/search?q=unknown+source+file+no+line+information), die diese `<unknown source file>`-Zeilen enthalten.

Lösungsvorschläge:
- Im Parser `<unknown source file>` erkennen und leeren Dateinamen an `Issue` übergeben
- Erstellen des Fingerprints robust machen: Es wird bereits bei `IOExceptions` auf einen Fallback-Fingerprint zurückgegriffen. Dieses Fallback könnte man auch für ungültige Dateinamen verwenden. (FullTextFingerprint#compute Z. 70)

Reply:
Das ist der nächste Bug, ja (den ersten habe ich ja noch schnell behoben). Ich befürchte, das kommt dann auch noch beim Kopieren der Files auf den Jenkins Master, usw. Ich werde den Fingerprinter (und ähnliche) auch noch robuster machen müssen. Evtl. lohnt es sich dann danach den Parser noch anzupassen. Aber erstmal ist das ein gutes Testobjekt für die restlichen Steps in meinem Plugin.

Sie brauchen da erst mal nichts mehr unternehmen, ich merge das dann sobald ich die Bugs bei mir behoben habe...